addons/web/static/src/legacy/js/views/abstract_model.js

```diff
@@ -104,7 +104,7 @@ var AbstractModel = mvc.Model.extend({
             value = fieldUtils.parse[field.type](value, field, {isUTC: true});
         } else if (field.type === 'selection' && value === false) {
             // process selection: convert false to 0, if 0 is a valid key
-            var hasKey0 = field.selection.find(option => option[0] === 0);
+            var hasKey0 = field.selection && field.selection.find(option => option[0] === 0);
```

Comment:
```suggestion
            var hasKey0 = field.selection?.find(option => option[0] === 0);
```
"Optional chaining" is the tool provided exactly for that matter :slightly_smiling_face: